CRM/SMS/Provider.php

```diff
@@ -79,9 +79,10 @@ public static function &singleton($providerParams = [], $force = FALSE) {
    * @param array $recipients
    * @param string $header
    * @param string $message
-   * @param int $dncID
+   * @param int $jobId
+   * @param int $sourceContactID
    */
-  abstract public function send($recipients, $header, $message, $dncID = NULL);
+  abstract public function send($recipients, $header, $message, $jobId = NULL, $sourceContactID = NULL);
```

Comment:
Based on a universe search I think this is more common 

![image](https://github.com/user-attachments/assets/a85ba8c9-0b91-4ed6-982e-15e5d4213bea)


```suggestion
  abstract public function send($recipients, $header, $message, $jobID = NULL, $userID = NULL);
```